rasa_sdk/interfaces.py

```diff
@@ -271,6 +272,29 @@ def add_slots(self, slots: List[EventType]) -> None:
             self.slots[event["name"]] = event["value"]
             self.events.append(event)
 
+    def get_intent_if_not_fallback(self) -> Optional[Text]:
+        """Retrieves the intent the last user message would have been classified as
+        if it was not nlu_fallback
+        """
+        last_user_action = self.get_last_event_for("user")
```

Comment:
I think `action` might be confusing within the Rasa Domain, as `action`s are usually something the assistant does.
```suggestion
        last_user_message = self.get_last_event_for("user")
```